web/sites/default/config/image.style.16_9_800w.yml

```diff
@@ -0,0 +1,17 @@
+uuid: 37a9dfee-8cdc-4b7e-a10f-de2bb2108c09
+langcode: en
+status: true
+dependencies:
+  module:
+    - focal_point
+name: 16_9_800w
+label: 16_9_800w
+effects:
+  c8babe8f-10c5-47bf-b72c-e5bb1f367845:
+    uuid: c8babe8f-10c5-47bf-b72c-e5bb1f367845
+    id: focal_point_scale_and_crop
```

Comment:
The change also seems to remove our responsive image style for the featured image/media and makes it a fixed size of 800x450 that gets sized only by CSS.
We talk a lot about how we try and deliver the best sized image based on device type. This may be okay if it somehow gives us retina-grade output, but it seems like a regression if we don't describe the change in the implementation details of the story.

Reply:
Reverted this change in branch [powr-3796c](https://github.com/eGovPDX/portlandor/pull/2166)